src/main/webapp/forms/array.addselect.jsp

```diff
@@ -48,12 +47,30 @@
             }
             out.println("</td>");
 
-            if (!isReadOnly) {
-                out.println("<td>&nbsp;</td><td><a href=\"javascript:popup('addselect', this, '" + auswahlherkunft + "', '" + datenfeld + "[" + i + "]', '');\">" + txt_newentry + "</a></td>");
-            }
             out.println("</tr>");
             i++;
         }
+
+        //Create new drop down list
```

Comment:
May the if(!isReadOnly) statement is missing here ? Or should the newItem Formular should be shown anyway ?

Reply:
No, it works all fine now